force-app/main/default/lwc/customEventLink/customEventLink.js

```diff
@@ -0,0 +1,72 @@
+import { LightningElement, track } from "lwc";
+import {
+  APPLICATION_SCOPE,
+  createMessageContext,
+  releaseMessageContext,
+  subscribe as subscribeMessageChannel,
+  unsubscribe as unsubscribeMessageChannel
+} from "lightning/messageService";
+import MC_FIRE_CUSTOM_EVENT from "@salesforce/messageChannel/msmxSheet__fireCustomEvent__c";
+/**
+ *
+ */
+const MESSAGE_CHANNELS = {
+  fireCustomEvent: MC_FIRE_CUSTOM_EVENT
+};
+/**
+ *
+ */
+export default class CustomEventLink extends LightningElement {
+  messageContext;
+
+  subscriptions = [];
+
+  @track record;
```

Comment:
`@track` decorator is not needed in latest LWC.
https://help.salesforce.com/s/articleView?id=release-notes.rn_lwc_track.htm&release=224&type=5